src/main/java/uk/gov/hmcts/befta/player/DefaultBackEndFunctionalTestScenarioPlayer.java

```diff
@@ -437,9 +484,18 @@ public void verifyTheResponseInTheContextWithAParticularSpecification(String res
     @Then("another call [{}] will get the expected response as in [{}]")
     public void performAndVerifyTheExpectedResponseForAnApiCall(String testDataSpec, String testDataId)
             throws IOException {
+        performAndVerifyTheExpectedResponseForAnApiCall(testDataSpec, testDataId, null);
+    }
+
+    private void performAndVerifyTheExpectedResponseForAnApiCall(String testDataSpec, String testDataId, String testDataReference)
+            throws IOException {
         BackEndFunctionalTestScenarioContext subcontext = new BackEndFunctionalTestScenarioContext();
         subcontext.initializeTestDataFor(testDataId);
-        this.scenarioContext.addChildContext(subcontext);
+        if (testDataReference == null) {
+            this.scenarioContext.addChildContext(subcontext);
+        } else {
+            this.scenarioContext.addChildContextByReference(testDataReference, subcontext);
```

Comment:
this method called here should be an overloaded one with the same name: addChildContext

Reply:
This has now been done.